src/main/resources/com/puppycrawl/tools/checkstyle/configuration_1_4.dtd

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
what is changed from 1_3 ?
Do we really need to bump version ? it is kind of strange that we change url in example - do not bump version.
and in the same time introduce 1_4.

We just need change files and copy them to new location.
Change all our XML files to point new location.
If user have old xml config that reference to old location of DTD - it should continue to work.
Files that we are packing and hosting should use new location.